### DISCUSSION TITLE
GCODE  out of machine bounds

### DISCUSSION THREAD

**Discussion by mommameg970** [arkypita/LaserGRBL#915] [Q&A]
I have my engraver opened in a different software... LASERWEB... 
I am trying so hard to get my machine back up and running... when i am in laserweb i am getting an alarm status saying GCODE out of machine bounds and i am 100% positive that is why my LaserGRBL software will not open at all... please help me fix this... :(
![image](https://user-images.githubusercontent.com/62154774/76665192-deedb500-6554-11ea-8d5e-75298b519b60.png)

**Comment by arkypita** [comment]
I can't help about laserweb

For LaserGRBL maybe you have a corrupted setting file.
Delete (or rename) LaserGRBL.Settings.bin in this folder: C:\Users\YourName\AppData\Roaming\LaserGRBL

AppData is an hidden folder, if you don't see it, simply type its name in address bar.
